apps/web/pages/api/basenames/[name]/getBasenameRegistrationPrice.ts

```diff
@@ -9,6 +8,7 @@ import {
 import { weiToEth } from 'apps/web/src/utils/weiToEth';
 import { formatWei } from 'apps/web/src/utils/formatWei';
 import { logger } from 'apps/web/src/utils/logger';
+import { CHAIN } from 'apps/web/pages/api/basenames/frame/constants';
```

Comment:
Nitpick: why define a constant? Also enforcing `base` means we can't test the frame on `baseSepolia`

Reply:
I pulled into a central definition it because it was hardcoded as `base` in a bunch of files, which made moving to `baseSepolia` error prone. 

And I don't love the auto-switching between `base` in prod and `baseSepolia` in development because in this case, the major things I need to test onchain require being in a prod / staging-prod environment. So I prefer to manually change the `CHAIN` definition to sepolia when I need to.